backends/nxp/backend/ir/tflite_optimizer/optimizer.py

```diff
@@ -164,9 +160,6 @@ def __init__(
             Optimization.COMBINE_HARD_SIGMOID_AND_MUL_INTO_HARD_SWISH: CombineHardSigmoidAndMulIntoHardSwish(
                 builder, conversion_config
             ),
-            Optimization.REPLACE_AVERAGE_POOL_BEFORE_FULLY_CONNECTED_WITH_SUM: ReplaceAveragePoolBeforeFullyConnectedWithSum(
```

Comment:
How is this related to the view support?

Reply:
Same here, not related to aten.view. Our fault. 
It is indeed a separate commit, and the PR descrition should have included the proper description: 
"Fix avegage pooling, which in case of kernel (1,1) turns into Sum in Neutron IR"

At least added in to the PR description.